### PR TITLE
include podman resource requirements for mac

### LINKED ISSUE #61: Add a note in the readme aboud ideal memory / CPU requirements for podman machine

### LINKED ISSUE #91: Kantra being stuck on mac x86
### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Konveyor version

N/A

### Priority

Major

### Current Behavior

When running `kantra analyze`, the analysis stops generating logs after a few minutes. The end of the log file does not show any failed steps or errors. Only two files are generated: `analysis.log` and `dependency.log` as output.

Potential solutions/reasons: 
1) The limit of open files in mounted volumes is 64, which is incredibly low (https://github.com/containers/podman/issues/16106#issuecomment-1317188581) Suggested by David 
2) Increase resources by changing podman machine settings. 

### Expected Behavior
```shell
$ ls ./output/ -1
analysis.log
dependencies.yaml
dependency.log
output.yaml
static-report
``` 
However, only two files are generated: `analysis.log` and `dependency.log` as output.

### How Reproducible

[Sometimes]

### Steps To Reproduce

Use this demo link to reproduce (https://github.com/konveyor/kantra#analyze-1)

### Environment

```markdown
- OS:
```

### Anything else?

Podman is running on macOS (Mac x86)

### PR DESCRIPTION
**Pull request by eemcmullan**
Closes #61 
Closes #91

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,25 @@ podman pull quay.io/konveyor/kantra:latest && podman run --name kantra-download
 podman machine init <vm_name> -v $HOME:$HOME -v /private/tmp:/private/tmp -v /var/folders/:/var/folders/
 ```
 
+Due to a known issue with limited number of open files in mounted volumes on MacOS, increase podman resources:
+
+```sh
+podman machine set <vm_name> --cpus 4 --memory 8192
```
Comment (pranavgaikwad) [inline README.md:25]: nit: 4 Gigs might be reasonable

**Review by pranavgaikwad — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jwmatthews) [reply README.md:25]: Technically, I don't believe the memory/cpu has a relationship to the number of open files.
Let's update this so we don't confuse ourselves/others in future.

The memory I believe will stop analysis of large applications, but it won't help for the open file limit

As to 4GB, that sounds reasonable, I ran with 8GB out of caution, i think @hhpatel14 is using 4GB.

File: README.md [hunk 2]
```diff
@@ -19,12 +19,25 @@ podman pull quay.io/konveyor/kantra:latest && podman run --name kantra-download
...
+```
+
+Prior to starting your podman machine, run
```
Comment (jwmatthews) [inline README.md:28]: Lets add the info about Mac and limiting the number of open files on a volume mount and please be sure to link to this known issue to help others to be fully aware:  https://github.com/containers/podman/issues/16106#issuecomment-1317188581

Note, I think this command needs to be run after every reboot, i.e. I don't think ulimit is persisted.
Worth calling this out so the end user is aware, they may want to put the ulimit in their shell login script

**Review by jwmatthews — CHANGES_REQUESTED** [review]
I suggested a few updates to consider.

**Review by jwmatthews — APPROVED** [review]
LGTM, thanks @eemcmullan

**Pull request merged**